draft-ppm-protocol.md

```diff
@@ -409,22 +419,31 @@ struct {
 
 ## Task Configuration {#task-configuration}
 
-Prior to the start of execution of the protocol, each participant must agree on
-the configuration for each task. A task is uniquely identified by its task ID:
+A task centers around evaluation of a specific Verifiable Distributed
+Aggregation Function (VDAF, see [VDAF]) over a set of client inputs. The VDAF
+determines the type of input measurements and the type of the output of the
+aggregation function. It also determines how many aggregators are involved (this
+document currently supports just two) and how many rounds of communication among
```

Comment:
Please file an issue for this rounds question. I am not convinced that we should commit to a fixed number of rounds as a protocol invariant.

Reply:
Before I do, I'd like to chat in person about the implication of this restriction and why I feel it's a safe assumption. Just to be clear: the intention is that the VDAF scheme determines the number of rounds, not the wrapper protocol (i.e., this specification).